app/main.py

```diff
@@ -1 +1,51 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+    def add_skills(self, skills: list) -> None:
+        for i in skills:
+            self.skills.append(i)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        super().add_skills(["JavaScript", "HTML", "CSS"])
```

Comment:
```suggestion
        self.skills.extend(["JavaScript", "HTML", "CSS"])
```


Please fix it here and other places